changeset.go

```diff
@@ -97,7 +97,7 @@ type Changeset struct {
 }
 
 // Upsert upserts resource in a context of a changeset
-func (cs *Changeset) Upsert(ctx context.Context, changesetNamespace, changesetName string, data []byte) error {
+func (cs *Changeset) Upsert(ctx context.Context, changesetNamespace, changesetName, id string, data []byte) error {
```

Comment:
I also think we can do better if we hid `changesetNamespace` and `changesetName` inside of `Changeset`.